gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/LocalGcsHelper.java

```diff
@@ -1,68 +0,0 @@
-/*
```

Comment:
Using `FakeStorageRpc` is fine (and probably a good idea for the reason you state) for the `gcs-nio` tests, but since the `LocalGcsHelper` would be helpful for other purposes outside of `gcs-nio`, I think we should still keep it.

Also, there's a PR in the master branch to rename `RemoteGcsHelper` to `RemoteStorageHelper` and `LocalGcdHelper` to `LocalDatastoreHelper`; in line with that, we should probably rename `LocalGcsHelper` to `LocalStorageHelper`.